espanol/huida/rechazar/arrastra/arrastra.md

```diff
@@ -0,0 +1 @@
+Al no poder mover las piernas, no queda más remedio que moverse con las manos. Saca fuerzas de donde no tiene y comienza a arrastrarase con sus brazos. Todos esos años de natación valieron la pena.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.